### ISSUE TITLE
amplify delete command doesn't really delete all files

### ISSUE THREAD

**Issue by gustavo-rios** [aws-amplify/amplify-cli#376]
When executing "amplify delete"  you receive a message that says: "Are you sure you want to continue?(This would delete all the project from the cloud and wipe out all the local amplify resource files)"

But I see that at least a couple of files are actually left in the project folder,  .amplifyrc  and src/aws-exports.js

This is using VS Code as my editor and Angular as the base project.

Just FYI

**Comment by kaustavghosh06** [comment]
@gustavo-rios Yes, as you can observe in our main README, the delete functionality is an experimental feature and we're waiting on user feedback on this like one to finalize on the feature and add enhancements to it. I'll mark this as an enhancement for our delete commands to delete the aws_exports file (or at least have a flag to do so).

**Comment by GeekEast** [comment]
@kaustavghosh06 HI, I also executed `amplify delete` but I found DynamoDB, S3 buckets and Cognito User Pool still exist. Hope `delete` could clear all resources related to that Amplify Project.

**Comment by traviswimer** [comment]
@GeekEast I'd check the Cloudformation logs (Amplify seems to be using Cloudformation stacks for all its resource creation/deletion). When I've created userpools with only Cloudformation (no Amplify), I've had issues with trying to delete them (I think because another resource depended on it).

Honestly though, I'm bothered by the fact that I actually CAN delete my userpools this way. It's a little scary to me that running `amplify delete` in the wrong environment could completely destroy all your user authentication data.

**Comment by kaustavghosh06** [comment]
@GeekEast Are you sure that `amplify delete` didn't get rid of the user-pool and the dynamo db tables? It should delete everything except S3 buckets (on which we set the "Retain") policy.

**Comment by kaustavghosh06** [comment]
Closing this in favor of a duplicate issue #81
